### ISSUE TITLE
Show Witsy icon in taskbar when hidden from status bar

### ISSUE THREAD

**Issue by arsaboo** [nbonamy/witsy#34]
Currently, when Witsy is minimized or hidden from the status bar, the application icon does not appear in the taskbar. This makes it difficult for users to access and manage the application quickly.

**Expected Behavior:**

The Witsy icon should be visible in the taskbar even when the application is minimized or hidden from the status bar.
Users should be able to easily click on the taskbar icon to bring the application back to the foreground.

**Comment by nbonamy** [comment]
It's supposed to work that way 😀 

What OS are you on?

**Comment by arsaboo** [comment]
Windows

**Comment by nbonamy** [comment]
BTW did you leave this option checked?

<img width="717" alt="image" src="https://github.com/user-attachments/assets/8cea50ac-cd2a-4b03-9148-3f498322e6f8" />

Also it is not hidden in the extended area?

<img width="136" alt="image" src="https://github.com/user-attachments/assets/4dc82ec4-3a3b-476d-bd24-fa7499e4b347" />

**Comment by arsaboo** [comment]
I may be confusing the terminology. My expectation was if we uncheck "Keep in Status Bar" and then X out Witsy, it should minimize to the notification area. 

Right now, with the screenshot you posted, if you X out of the app, Witsy will close. What I was looking for is some way to minimize to the notification area (the second screenshot you posted).

**Comment by arsaboo** [comment]
![image](https://github.com/user-attachments/assets/eaad9990-9c76-48e5-ad99-6ef257468079)
I still don't see in the notification area even with "Keep in Status bar" checked.

**Comment by nbonamy** [comment]
The Witsy icon is the lightbulb icon.

**Comment by arsaboo** [comment]
Oops....I guess I was looking for the Witsy icon. Closing it.

**Comment by nbonamy** [comment]
True that on Windows I should probably use the Witsy icon. Does not fit macOS but could definitely fit on Windows. Will look into it.

**Comment by nbonamy** [comment]
Icon updated in just released 1.32.0

**Comment by arsaboo** [comment]
Now, it is blank (after it updated):

![image](https://github.com/user-attachments/assets/1ec5998d-8175-4020-9c9d-b6433adf51c3)

**Comment by nbonamy** [comment]
Yes tried to cancel 1.32.0 but obviously too late 😀 Should be fixed in upcoming 1.32.1

**Comment by arsaboo** [comment]
The icon did not show up today (it was there yesterday):
![image](https://github.com/user-attachments/assets/5fc063ae-2179-436e-9711-7d99a749d9d4)

**Comment by nbonamy** [comment]
Are you running 1.32.1?

<img width="458" alt="image" src="https://github.com/user-attachments/assets/495e3fd5-c3f3-4ee5-a23c-708978484f93" />

**Comment by arsaboo** [comment]
In the About section, it showed 1.32.0 (even though I was able to add OpenRouter API). I reinstalled the app and now see the icon in the notification area. 

I will keep the issue open for now, but it may be a temporary glitch.

**Comment by nbonamy** [comment]
I checked just released 1.33.0 and it was OK on my side. Can you confirm? Thanks.
